app/src/main/java/id/rizmaulana/covid19/ui/detail/DetailActivity.kt

```diff
@@ -75,6 +78,14 @@ class DetailActivity : BaseActivity() {
         }
     }
 
+    private fun showItemListDialog(dataContext: CovidDetail) {
+        AlertDialog.Builder(this)
+            .setItems(arrayOf("Pin to Dashboard")) { dialog, which ->
```

Comment:
How about we move the `arrayOf("Pin to Dashboard")` to strings.xml as `string-array`
and change it like this `resources.getStringArray(R.array.options)` ? 🤔

Reply:
yes its better, will do